addons/binding/pom.xml

```diff
@@ -72,6 +72,7 @@
     <module>org.openhab.binding.toon</module>
     <module>org.openhab.binding.urtsi</module>
     <module>org.openhab.binding.vitotronic</module>
+    <module>org.openhab.binding.connectsdk</module>
```

Comment:
The module must be in alphabetical order.

Reply:
OK